core/src/main/java/com/google/cloud/sql/core/InternalConnectorRegistry.java

```diff
@@ -191,6 +191,16 @@ public ConnectionMetadata getConnectionMetadata(ConnectionConfig config) {
     return getConnector(config).getConnection(config).getConnectionMetadata(connectTimeoutMs);
   }
 
+  /** Internal use only: Force refresh the connection info. */
+  public void forceRefresh(ConnectionConfig config) {
```

Comment:
Shall we put some kind of unit test around this?

Reply:
Added.